docs/get-started/getting-started-on-prem.md

```diff
@@ -296,6 +299,32 @@ $SPARK_HOME/bin/spark-shell \
        --jars ${SPARK_CUDF_JAR},${SPARK_RAPIDS_PLUGIN_JAR}
 ```  
 
+### MIG GPU on YARN
```

Comment:
Can we add a link to some information about what MIG is, for example https://docs.nvidia.com/datacenter/tesla/mig-user-guide/index.html#introduction ?